tests/test_typed_comments.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from black.nodes import is_type_comment
+from blib2to3.pgen2 import token
+from blib2to3.pytree import Leaf
+
+
+@pytest.mark.parametrize(
+    "comment_str",
+    ["# type: int", "#  type: int", "#   type: int", "# type : int", "# type  : int"],
+)
+def test_typed_comments(comment_str: str) -> None:
+    assert is_type_comment(Leaf(token.COMMENT, comment_str))
```

Comment:
Would be good to have tests for `test_not_typed_comment`

Reply:
> Would be good to have tests for `test_not_typed_comment`

for example
` # this is a comment about a :param called type`

Reply:
Thanks for the suggestion! Added!